src/components/ClinicRegularDetails.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import '../App.css';
+
+export interface OpeningHours {
+  monday: { opensAt: string; closesAt: string } | {};
+  tuesday: { opensAt: string; closesAt: string } | {};
+  wednesday: { opensAt: string; closesAt: string } | {};
+  thursday: { opensAt: string; closesAt: string } | {};
+  friday: { opensAt: string; closesAt: string } | {};
+  saturday: { opensAt: string; closesAt: string } | {};
+  sunday: { opensAt: string; closesAt: string } | {};
```

Comment:
I guess the default object means closed that day? Is that better than making it optional?

Reply:
https://context.reverso.net/%C3%BCbersetzung/englisch-deutsch/closes+at

Reply:
I was talking about the `| {}` part

Reply:
lol just saw its irrelevant. previously we used regular opening hours. Now just emergency o.h.